packages/data-broker/src/DataBroker/mergeAnalytics.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { Analytic, AnalyticResults as RawAnalyticResults } from '../types';
+
+export interface AnalyticResults {
+  results: {
+    analytics: Analytic[];
+    numAggregationsProcessed: number;
+  }[];
+  metadata: {
+    dataElementCodeToName: Record<string, string>;
+  };
+}
+
+export const mergeAnalytics = (
```

Comment:
This is just the existing `DataBroker.mergeAnalytics()` extracted in a pure function